app/websocket/websocket_app.py

```diff
@@ -62,18 +50,29 @@ def is_this_an_opengrid_node():
     socketio.emit("/identity", "Websocket OpenGrid")
 
 
-@socketio.on("/cmd")
-def cmd(message):
-    try:
-        worker = _maybe_create_worker("worker", "grid")
+@socketio.on("/set-grid-id")
+def set_grid_name(msg):
+    me = hook.local_worker
+    me.id = msg["id"]
+    me.is_client_worker = False
 
-        worker.verbose = True
-        sy.torch.hook.local_worker.add_worker(worker)
 
-        response = _request_message(worker, message)
+@socketio.on("/connect-node")
+def connect_node(msg):
+    try:
+        new_worker = gr.WebsocketGridClient(hook, msg["uri"], id=msg["id"])
+        new_worker.connect()
+        know_nodes[msg["id"]] = new_worker
+        socketio.emit("/connect-node", "Succefully connected!")
+    except Exception as e:
+        socketio.emit("/connect-node", str(e))
 
-        _store_worker(worker, "worker")
 
+@socketio.on("/cmd")
+def cmd(message):
+    try:
+        worker = hook.local_worker
```

Comment:
We want to keep a version of the worker stored on the database (in case we lost connection). But we can keep it like this for now since we probably want to use postgres any way
#235

Reply:
I'll open an issue for tensor persist integration with apps.